include/class.filter.php

```diff
@@ -278,8 +278,14 @@ function getActions() {
      * If the matches() method returns TRUE, send the initial ticket to this
      * method to apply the filter actions defined
      */
-    function apply(&$ticket, $vars) {
+    function apply(&$ticket, $vars, $postCreate=false) {
         foreach ($this->getActions() as $a) {
+            //control when certain actions should be applied
+            //if action is send email and postCreate == false, skip
+            //if action is not send email and postCreate == true, skip
+            if ((($a->type == 'email') ? !$postCreate : $postCreate))
```

Comment:
Did you mean - at least based on the comment above.
```php
 if ($a->type == 'email' && !$postCreate)
     continue;
```

Reply:
I wanted to write it in the shortest way possible. Originally, I had this:

```
if (($a->type == 'email' && !$postCreate) || ($a->type != 'email' && $postCreate))
     continue;
```

I tested the current code a lot, and both give the same results. Should I still just write it out for readability? I wrote the comment above it because I kept second guessing it.